src/components/AboutModal.vue

```diff
@@ -70,7 +71,7 @@ export default {
   },
   data: () => ({
     appVersion: appInfo.version,
-    sdkVersion: sdkInfo.version,
+    sdkVersion: packageLock.packages['node_modules/@aeternity/aepp-sdk'].version,
     contractsVersion: contractInfo.version,
     compilerUrl: process.env.VUE_APP_COMPILER_URL,
```

Comment:
how do we do the deployment of new pairs now? if we provide pre-compiled bytecode (which we should) the compiler url won't be needed

Reply:
I removed the the compiler URL info

Reply:
also switched to the `package.json` sdk version and removed the package.json dependency from AboutModal